x-pack/test/apm_api_integration/tests/alerts/chart_preview.spec.ts

```diff
@@ -115,5 +115,54 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         )
       ).to.equal(true);
     });
+
+    it('transaction_duration with transaction name', async () => {
+      const options = {
+        params: {
+          query: {
+            start,
+            end,
+            serviceName: 'opbeans-java',
+            transactionName: 'DispatcherServlet#doGet',
+            transactionType: 'request',
+            environment: 'ENVIRONMENT_ALL',
+            interval: '5m',
+          },
+        },
+      };
+      const response = await apmApiClient.readUser({
+        ...options,
+        endpoint: 'GET /internal/apm/rule_types/transaction_duration/chart_preview',
+      });
+
+      expect(response.status).to.be(200);
+      expect(response.body.latencyChartPreview[0].data[0]).to.eql({
+        x: 1627974600000,
+        y: 18485.85714285714,
```

Comment:
Ah, these are archives. We should replace it with Synthtrace at some point, but at least we capture the current value so we don't unintentionally change it.

Reply:
We might be able to tackle it during test plan for 8.8. I've created an issue to capture this and the other tests that still rely on archives: https://github.com/elastic/kibana/issues/155359